examples/server.c

```diff
@@ -108,7 +108,9 @@ int main(int argc, char** argv) {
     signal(SIGINT, stopHandler); /* catches ctrl-c */
 
     UA_ServerNetworkLayer nl = UA_ServerNetworkLayerTCP(UA_ConnectionConfig_standard, 16664);
-    UA_ServerConfig config = UA_ServerConfig_standard;
+    UA_ServerConfig config;
+    UA_ServerConfig_standard_new(&config);
```

Comment:
Since this adds a whole lot of noise to the PR how about i open a separate one just for the config changes. Then we can first check that and merge it as a prerequisite for this PR?